tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+
+1. `git clone https://github.com/google/digitalbuildings.git`
+2. `cd digitalBuildings/tools`
+3. Depending on your operating system run either of the global setup scripts to configure dependencies
+    * Windows: `pip_install.bat`
+    * Unix/MacOs: `sudo bash pip_install.sh`
+5. Create a directory for ABEL to use at `~/abel/` and navigate to it
+    * This will be used for storing authentication credentials and documents output from ABEL.
+4. Obtain a spreadsheet token for your google sheets account
+    1. Be added to a GCP service account with `Service Account User` and `Service Account Token Creator` roles.
+        * Please contact your IoT TPM for details and to be added to a project.
+    2. Create a token request file at `~/abel/token_request.json` with contents:
+        ```
+        {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+        ```
+    3. Run the below curl command to obtain a token:
+        ```
+            curl -X POST \
+            -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) \
+            -H "Content-Type: application/json; charset=utf-8" \
+            -d @token_request.json \
+            "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/your-service-account@project-id.iam.gserviceaccount.com:generateAccessToken" >| spreadsheet_token.json
+        ```
+    * `spreadsheet_token.json` should look something like:
+        ```
+        {
+          "accessToken": "ya29.c.AXv0zTOa9p5K78GtGntoIQCqyt2R572qZ3…",
+          "expireTime": "2022-02-24T19:57:26Z"
+        }
+        ```
+
+## The ABEL Spreadsheet
+The ABEL spreadsheet is what serves as a frontend for ABEL.
+
+Please see the [spreadsheet docs](./validators/README.md) for detailed documentation.
+
+## Using ABEL
+The core functionality of ABEL is to:
+* Ingest a ABEL spreadsheet and export a valid Building Config document
+* Ingest a Building config and export an ABEL spreadsheet
+
+#### Command-line arguments for ABEL:
+* `-t / --token` (required) path to the GCP project token. The path should be ~/abel/token.json 
+* `-s / --spreadsheet_id` (required) id for a google sheets spreadsheet. If necessary, create a blank spreadsheet. 
+    * [Google Sheets Template](https://docs.google.com/spreadsheets/d/1qKMlpJI5-_h_8innNniEkpatMBcRHSGekrRwTsPQ618/edit#gid=455639674)
+
+    * **MAKE SURE TO SHARE THE SPREADSHEET WITH YOUR SERVICE ACCOUNT AND PROJECT ID AS AN EDITOR**
+
+* `-b / --building_config` (optional) absolute path to a local building configuration file
+    * [View a sample building config](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md)
+    
+#### Spreadsheet -> Building Config
+```
+python3 abel.py -t absolute/path/to/token -s input_spreadsheet_id
```

Comment:
add the arguments in `<input_spreadsheet_id>` ?

Reply:
Done.